### ISSUE TITLE
Create documentation for the Plugin example 

### ISSUE THREAD

**Issue by cmoulliard** [halkyonio/operator#214]
## Documentation

Create documentation, how to instructions for the `Plugin example`

**Comment by metacosm** [comment]
Isn't this done, @cmoulliard?

**Comment by cmoulliard** [comment]
Yes. We will close it and maybe open another ticket to re-org the doc under `operator-framework`
